### PR TITLE
Watt this in situ carriage block lubrication

### PR DESCRIPTION
**Pull request by jpeterse**
### Which mods are added by this PR?
a mod that allow lubrication of the linear rail guide blocks while install in the printer.

### How was it tested?
made the mod to my own 2.4r2 printer

### Any background context you want to provide?
With the speed of the Voron printers print at, the distance traveled quick add up. Which means the manufacturers recommended lubrication intervals, comes up sooner than ever before. This mod allow for lubricating the guide blocks without any disassembly of the printer.

### Screenshots (if appropriate)
Several pictures are included in the submission, including a short video demonstrating the concept.

### ⚠️⚠️**PLEASE READ AND COMPLETE THE CHECKLIST BELOW**⚠️⚠️

  * [X ] I have read the rules available [here](https://github.com/VoronDesign/VoronUsers/wiki/Mod-Submission-Rules) and
        my mod adheres to these rules.
  * [ X] This mod was created by myself and I agree to publish it under the repository
        [license](https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md)

### CODE REVIEW

File: printer_mods/watt-this/in-situ_linear_rail_greasing/.metadata.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# A short description of your mod. Try to describe your mod with 1-2 short sentences
+description: A modification that allow lubrication of the linear reals while fully assembled.
+# Mod version: Set this to 1 when you are submitting a new mod. When you make changes to your mod, you may increment this number
+mod_version: 1
+# You may add your discord username here so people can contact you about your mod
+discord_username: Jan_P
+# The CAD files for your mod. Please provide them as a list.
+# Note that each mod must include at least one CAD file.
+cad:
+- greasing_concept.step
```
Comment (MapleLeafMakers) [inline printer_mods/watt-this/in-situ_linear_rail_greasing/.metadata.yml:11]: Files listed in the metadata need to include the path to that file, this should be:
` - CAD/greasing_concept.step`

File: printer_mods/watt-this/in-situ_linear_rail_greasing/.metadata.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# The image files provided with your mod. These may be instructions you use in your README.md file, or other additional pictures.
+# **Note that the first image specified here will be used as cover image on mods.vorondesign.com**
+# All paths are relative to this .metadata.yml file.
+images:
```
Comment (MapleLeafMakers) [inline printer_mods/watt-this/in-situ_linear_rail_greasing/.metadata.yml:15]: All the files listed here should be prefixed with `IMG/`

**Review by MapleLeafMakers — CHANGES_REQUESTED** [review]
The .webp files here appear to be redundant, as there are already png versions of them all. can the .webp files be removed?

**Comment by FHeilmann** [comment]
@jpeterse I see you have added a video to your PR, this video is rather large. Would it be possible to reduce that video size a bit? The reason for my request is that VoronUsers is already ~10GB for a single checkout and we'd like to try and keep the overall size as small as possible. As an alternative, you can host the video unlisted on youtube and add a link to your readme to that instead.

**Comment by jpeterse** [comment]
Moved the video as requested

**Pull request merged**